core/email_api.php

```diff
@@ -1555,3 +1555,29 @@ function email_shutdown_function() {
 		email_send_all();
 	}
 }
+
+/**
+ * Get the list of supported email actions.
+ *
+ * @return array List of actions
+ */
+function email_get_actions() {
+	$t_actions = array( 'updated', 'owner', 'reopened', 'deleted', 'bugnote' );
+
+	if( config_get( 'enable_sponsorship' ) == ON ) {
+		$t_actions[] = 'sponsor';
+	}
+
+	$t_actions[] = 'relation';
+
+	$t_statuses = MantisEnum::getAssocArrayIndexedByValues( config_get( 'status_enum_string' ) );
+	ksort( $t_statuses );
+	reset( $t_statuses );
+
+	foreach( $t_statuses as $t_status => $t_label ) {
```

Comment:
Yep, I removed it.  It was used in one case of the old code and not in the other.  I generally don't mind including it for readability, but will remove it.